packages/ui/src/stories/icons.stories.tsx

```diff
@@ -19,7 +16,7 @@ const IconBlock: React.FC<IconBlockProps> = ({ name, icon, iconProps }) => {
     const Icon = icon
 
     return (
-        <Space prefixCls='condo-space' direction='vertical' size={8} style={{ width: 120 }} align='center'>
+        <Space direction='vertical' size={8} width={120} align='center'>
             <div style={{ fontSize: 32 }}>
```

Comment:
constant

Reply:
It is a storybook file. IMO this is unnecessary